### DISCUSSION TITLE
Documenting DBT with ChatGPT feat chaptgpt-md

### DISCUSSION THREAD

**Discussion by Nintorac** [bramses/chatgpt-md#21] [Show and tell]
Hey, hope you enjoy!

https://www.nintoracaudio.dev/data-eng/2023/03/18/dbt-generator.html

**Comment by bramses** [comment]
Love this, thank you! Please feel free to PR the [templates repo](https://github.com/bramses/chatgpt-md-templates) if you have a DBT template you'd like to share

**Reply by Nintorac** [reply]
So far it only covers dbt `sources` documentation and there's quite a bit of work to incorporate the rest and fit it in the context so not sure if it's too suitable in the current state and don't have the motivation to work on that for now.

Happy to PR just the sources prompt though if you think it's worth it.

FYI came across this repo yesterday https://github.com/f/awesome-chatgpt-prompts (via https://github.com/citiususc/Smarty-GPT). Could be worth sourcing templates from there?

**Comment by bramses** [comment]
I really like this line especially:

> One fun side effect of this format over using the ChatGPT website is that you can modify the assistant replies in place which can help to fix small mistakes in real time while larger changes can be achieved via further conversation.

I do the same thing with my ChatGPT MD notes!!
